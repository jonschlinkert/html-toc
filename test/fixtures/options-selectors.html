<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>
<body>
  <div id="toc"></div>
  <h1>ABC</h1>
  <h2>Foo</h2>
  <h3>Baz</h3>
  <h2>Bar</h2>
  <h3>Baz</h3>
  <h2>Baz</h2>
  <h3>Baz</h3>
</body>
</html>
