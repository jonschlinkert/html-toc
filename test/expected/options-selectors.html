<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Document</title>
</head>
<body>
  <div id="toc"><ul class="nav sidenav"><li><a href="#abc">ABC</a><ul class="nav"><li><a href="#abc-foo">Foo</a><ul class="nav"><li><a href="#abc-foo-baz">Baz</a></li></ul></li>
<li><a href="#abc-bar">Bar</a><ul class="nav"><li><a href="#abc-bar-baz-1">Baz</a></li></ul></li>
<li><a href="#abc-baz-2">Baz</a><ul class="nav"><li><a href="#abc-baz-2-baz-3">Baz</a></li></ul></li></ul></li></ul></div>
  <h1 id="abc">ABC<a href="#abc" name="abc" class="anchor">
  <span class="anchor-target" id="abc"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h1>
  <h2 id="abc-foo">Foo<a href="#abc-foo" name="abc-foo" class="anchor">
  <span class="anchor-target" id="abc-foo"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h2>
  <h3 id="abc-foo-baz">Baz<a href="#abc-foo-baz" name="abc-foo-baz" class="anchor">
  <span class="anchor-target" id="abc-foo-baz"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h3>
  <h2 id="abc-bar">Bar<a href="#abc-bar" name="abc-bar" class="anchor">
  <span class="anchor-target" id="abc-bar"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h2>
  <h3 id="abc-bar-baz-1">Baz<a href="#abc-bar-baz-1" name="abc-bar-baz-1" class="anchor">
  <span class="anchor-target" id="abc-bar-baz-1"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h3>
  <h2 id="abc-baz-2">Baz<a href="#abc-baz-2" name="abc-baz-2" class="anchor">
  <span class="anchor-target" id="abc-baz-2"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h2>
  <h3 id="abc-baz-2-baz-3">Baz<a href="#abc-baz-2-baz-3" name="abc-baz-2-baz-3" class="anchor">
  <span class="anchor-target" id="abc-baz-2-baz-3"></span>
  <span class="glyphicon glyphicon-link"></span>
</a></h3>
</body>
</html>
